<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwin</class>
 <widget class="QDialog" name="mainwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sounds and Patterns</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,4,4,0,1">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_port">
       <property name="text">
        <string>Seleziona porta seriale</string>
       </property>
       <property name="buddy">
        <cstring>comboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openSerial_btn">
       <property name="text">
        <string>Connetti Arduino</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="box_sound">
     <property name="title">
      <string>Suoni</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="snd_btn01">
        <property name="text">
         <string>Ambulanza</string>
        </property>
        <property name="iconSize">
         <size>
          <width>56</width>
          <height>56</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="snd_btn02">
        <property name="text">
         <string>Campanello</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="snd_btn03">
        <property name="text">
         <string>Allarme</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="snd_btn04">
        <property name="text">
         <string>Fischietto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="snd_btn05">
        <property name="text">
         <string>Clacson</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="snd_btn06">
        <property name="text">
         <string>Elettrodomestico</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="box_pattern">
     <property name="title">
      <string>Pattern</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pattern_btn01">
        <property name="text">
         <string>Ambulanza</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pattern_btn02">
        <property name="text">
         <string>Campanello</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pattern_btn03">
        <property name="text">
         <string>Allarme</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pattern_btn04">
        <property name="text">
         <string>Fischietto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pattern_btn05">
        <property name="text">
         <string>Clacson</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pattern_btn06">
        <property name="text">
         <string>Elettrodomestico</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Suono</string>
       </property>
       <property name="buddy">
        <cstring>comboSuoni</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSuoni">
       <item>
        <property name="text">
         <string>Ambulanza</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Campanello</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Allarme</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fischietto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clacson</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Elettrodomestico</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
